src/components/textfieldControl/js/TextfieldSlider.js

```diff
@@ -0,0 +1,189 @@
+        },
+        listeners: {
+            "onCreate.addContainerStyle": {
+                "this": "{that}.container",
+                method: "addClass",
+                args: ["{that}.options.styles.container"]
+            }
+        },
+        distributeOptions: [{
+            source: "{that}.options.scale",
+            target: "{that > fluid.textfield > controller}.options.scale"
+        }]
+    });
+
+    fluid.defaults("fluid.slider", {
+        gradeNames: ["fluid.viewComponent"],
+        modelRelay: {
+            target: "value",
+            singleTransform: {
+                type: "fluid.transforms.stringToNumber",
+                input: "{that}.model.stringValue"
+            }
+        },
+        invokers: {
+            setModel: {
+                changePath: "stringValue",
+                value: {
+                    expander: {
+                        "this": "{that}.container",
+                        "method": "val"
+                    }
+                }
+            },
+            updateSliderAttributes: {
+                "this": "{that}.container",
+                method: "attr",
+                args: [{
+                    "min": "{that}.model.range.min",
+                    "max": "{that}.model.range.max",
+                    "step": "{that}.model.step",
+                    "type": "range",
+                    "value": "{that}.model.value",
+                    "aria-labelledby": "{that}.options.ariaOptions.aria-labelledby",
+                    "aria-label": "{that}.options.strings.aria-label"
+                }]
+            }
+        },
+        listeners: {
+            // "onCreate.initSliderAttributes": {
+            //     "this": "{that}.container",
+            //     method: "attr",
+            //     args: [{
+            //         "min": "{that}.options.range.min",
+            //         "max": "{that}.options.range.max",
+            //         "step": "{that}.options.step",
+            //         "type": "range",
+            //         "value": "{that}.model.value",
+            //         "aria-labelledby": "{that}.options.ariaOptions.aria-labelledby",
+            //         "aria-label": "{that}.options.strings.aria-label"
+            //     }]
+            // },
+            "onCreate.initSliderAttributes": "{that}.updateSliderAttributes",
+            "onCreate.bindSlideEvt": {
+                "this": "{that}.container",
+                "method": "on",
+                "args": ["input", "{that}.setModel"]
+            },
+            "onCreate.bindRangeChangeEvt": {
+                "this": "{that}.container",
+                "method": "on",
+                "args": ["change", "{that}.setModel"]
+            }
+        },
+        modelListeners: {
+            "value": [{
+                "this": "{that}.container",
+                "method": "val",
+                args: ["{change}.value"],
+                // If we don't exclude init, the value can get
```

Comment:
Might be helpful for the future maintenance by reducing the repetition of this same comment. Use one at the start of the `modelListeners` block?